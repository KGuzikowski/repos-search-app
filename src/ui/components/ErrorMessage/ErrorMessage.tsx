import styled from 'styled-components'

const ErrorMessage = styled.div`
    background: #ffb3b3;
    color: #ff0000;
    margin: 30px;
    font-size: 20px;
    padding 10px 20px;
`

export default ErrorMessage