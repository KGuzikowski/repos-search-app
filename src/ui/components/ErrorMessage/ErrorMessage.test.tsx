import React from 'react'
import { create } from 'react-test-renderer'
import ErrorMessage from './ErrorMessage'

it('renders ErrorMessage component correctly', () => {
    const component = create(<ErrorMessage>Testing</ErrorMessage>)
    expect(component.toJSON()).toMatchSnapshot()
})