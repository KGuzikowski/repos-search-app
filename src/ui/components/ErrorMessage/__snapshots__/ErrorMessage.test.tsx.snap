// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ErrorMessage component correctly 1`] = `
<div
  className="sc-AxjAm dyTfnv"
>
  Testing
</div>
`;
